go/v1beta1/storage/elasticsearch.go

```diff
@@ -448,6 +442,7 @@ func (es *ElasticsearchStorage) CreateOccurrence(ctx context.Context, projectID,
 	if o.CreateTime == nil {
 		o.CreateTime = ptypes.TimestampNow()
 	}
+	o.Name = fmt.Sprintf("projects/%s/occurrences/%s", projectID, uuid.New().String())
```

Comment:
I think the reasoning here make sense, but just to be clear this means that collectors will not be able to set the name of new occurrences? It'll be overwritten by the elasticsearch backend